cis/schema.json

```diff
@@ -233,15 +243,15 @@
                     "lastUsed": {
                         "type": "string",
                         "format": "date-time",
-                        "description": "UNKNOWN"
+                        "description": "Date at which this group membership last granted access to a resource (generally, an RP) for the user. This may be used to automatically expire and deny access when the lastUsed attribute is older than 3 month, for example."
                         },
                     "name": {
                         "type": "string",
-                        "description": "Name of the group, as given by the IdP."
+                        "description": "Name of the group."
                         },
                     "uuid": {
                         "type": "string",
-                        "description": "Reserved; profile providers should not use this value."
+                        "description": "Unique identifier for the authoritative group. Profile consumers should not try to interpret this value in any way."
```

Comment:
If I were building a document editing tool and wanted to be able to associate groups with access to a document, would this be the right value to store in my access-control table, rather than name?